src/libcollections/vec.rs

```diff
@@ -1296,6 +1296,17 @@ impl<T: PartialEq> Vec<T> {
     pub fn dedup(&mut self) {
         self.dedup_by(|a, b| a == b)
     }
+    
```

Comment:
This line has trailing whitespace which will need to be removed, that's why travis failed